navigation_service_node:
  ros__parameters:

    # ROS2 Navigation Service Parameters
    # The following parameters are used to configure the ROS2 Navigation Service.

    navigate_to_pose_service_name: "send_robot"
    navigate_to_pose_timeout: 10.0 # seconds
    navigate_to_pose_update_rate: 10.0 # Hz
    navigate_to_pose_position_tolerance: 0.5 # meters
    navigate_to_pose_orientation_tolerance: 0.25 # radians

    goal_pose_topic_name: "goal_pose"
    goal_pose_topic_queue_size: 10

    robot_source_frame_name: "base"
    robot_target_frame_name: "map"